### ISSUE TITLE
Scaffold issue with absolute URLs ( Patch included )

### ISSUE THREAD

**Issue by alimoosavi15** [SocialEngine/phpv4-issues#275]
Scaffold tries to find the absolute path within the local directories.

To produce the issue add the following to application/themes/default/theme.css

@font-face {
  font-family: 'Chewy';
  src: local('Chewy'), url('http://fonts.gstatic.com/s/chewy/v7/rb3O4cUMVLYzfgbaJOdJHw.woff2') format('woff2');
}

application/libraries/Scaffold/modules/Absolute_Urls/Absolute_Urls.php ( Line 72 )

The problem :

```php
if( substr($url, 0, 5) == "data:" ) { // Definitely skip this
```

Fixed :
```php
if( substr($url, 0, 5) == "data:" || substr($url, 0, 7) == "http://" ) { // Definitely skip this

**Comment by shivani2910** [comment]
Hi @alimoosavi15, 

Thanks for reaching out and sorry for the delay in responding to this one. We've forwarded this issue to our team and also provided the fix that you've mentioned. Our team will look into this and make necessary changes in the code in the next version.

Thanks,
Shivani

**Comment by shivani2910** [comment]
Fixed in [4.8.10](http://blog.socialengine.com/2016/01/20/socialengine-php-4-8-10-is-released/)
